openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,711 @@
+            rigidWater=self._settings.system_settings.rigid_water,
+        )
+
+        stateA_topology = stateA_modeller.getTopology()
+
+        # TODO: Algorithm this magic number
+        # We center the system by adding (14, 14, 14) [box is ~ 28.6 A per side]
+        stateA_positions = stateA_modeller.getPositions() + np.array([14, 14, 14]) * unit.angstrom
+
+        # Remove the ligand from state A and replace with state B ligand
+        stateB_topology = _rbfe_utils.append_new_topology_item(
+            stateA_topology,
+            stateB_openff_ligand.to_topology().to_openmm(),
+            exclude_residue_name=stateA_openff_ligand.name,
+        )
+
+        # Create the state B System
+        smirnoff_stateB = SMIRNOFFTemplateGenerator(
+            forcefield=self._settings.topology_settings.forcefield['ligand'],
+            molecules=[stateB_openff_ligand],
+        )
+        omm_forcefield_stateB = app.ForceField(
+            *[ff for (comp, ff) in self._settings.topology_settings.forcefield.items()
+              if not comp == 'ligand']
+        )
+        omm_forcefield_stateB.registerTemplateGenerator(smirnoff_stateA.generator)
+        stateB_system = omm_forcefield_stateB.createSystem(
+            stateB_topology,
+            nonbondedMethod=nonbonded_method,
+            nonbondedCutoff=self._settings.system_settings.nonbonded_cutoff,
+            constraints=constraints,
+            rigidWater=self._settings.system_settings.rigid_water,
+        )
+
+        # Define mappings between the two systems
+        ligand_mappings = _rbfe_utils.topologyhelpers.get_system_mappings(
+            self._mapping.mol1_to_mol2,
+            stateA_system, stateA_topology, stateA_openff_ligand.name,
+            stateB_system, stateB_topology, stateB_openff_ligand.name,
+            # TODO: Are these settings?
+            fix_constraints=True,
+            remove_element_changes=True,
+        )
+
+        stateB_positions = _rbfe_utils.topologyhelpers.set_and_check_new_positions(
+            ligand_mappings, stateA_topology, stateB_topology,
+            insert_positions=stateB_openff_ligand.conformers[0],
+            # TODO: Remove this magic number
+            shift_insert=np.array([14, 14, 14]),
+        )
+
+        alchem_settings = self._settings.alchemical_settings
+        hybrid_factory = _rbfe_utils.relative.HybridTopologyFactory(
+            stateA_system, stateA_positions, stateA_topology,
+            stateB_system, stateB_positions, stateB_topology,
+            old_to_new_atom_map=solvent_mappings['old_to_new_atom_map'],
+            old_to_new_core_atom_map=solvent_mappings['old_to_new_core_atom_map'],
+            use_dispersion_correction=alchem_settings.use_dispersion_correction,
+            softcore_alpha=alchem_settings.softcore_alpha,
+            softcore_LJ_v2=alchem_settings.softcore_LJ_v2,
+            # TODO: Is this setting missing?
+            softcore_LJ_v2_alpha=0.85,
```

Comment:
```suggestion
            softcore_LJ_v2_alpha=alchem_settings.softcore_alpha,
```

Makes not sense to have both a normal and a v2 softcore alpha if you're only going to use one. So I just squashed them into one. If folks want to use the old version they'll have to manually change the default.